ArgsParsing/Types/HexColor.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace ArgsParsing.Types
+{
+    /// <summary>
+    /// Wrapper around a color in hexadecimal string notation, e.g. <c>FF0000</c> for pure red.
+    /// The string is _not_ prefixed with a hash symbol.
+    /// </summary>
+    public class HexColor
+    {
+        public string HexColorString { get; }
+
+        public HexColor(string hexColorString)
+        {
+            HexColorString = hexColorString;
+        }
+
+        public static implicit operator string(HexColor hexColor) => hexColor.HexColorString;
```

Comment:
hashtag in front? idk if that's better than no hashtag though, but it makes it clear it's a hex colour

Reply:
I don't know which is better either, but for _some_ guidance, I could make it follow TwitchLib's convention, which includes the hash